webhooks/webhook_salient/main.py

```diff
@@ -122,13 +129,13 @@ def do_POST(self):
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(
-        description="Forward Stream Webhook Events to NX VMS as Alerts."
+        description="Forward Stream Webhook Events to CompleteView VMS as Events."
     )
-    parser.add_argument("--username", help="NX VMS Username.", required=True)
-    parser.add_argument("--password", help="NX VMS Password.", required=True)
-    parser.add_argument("--vms", help="VMS API Endpoint.", required=True)
+    parser.add_argument("--username", help="Recording Server Username.", required=True)
+    parser.add_argument("--password", help="Recording Server Password.", required=True)
+    parser.add_argument("--vms", help="Recording Server API Endpoint.", required=True)
```

Comment:
```suggestion
    parser.add_argument("--username", help="Salient VMS Username.", required=True)
    parser.add_argument("--password", help="Salient VMS Password.", required=True)
    parser.add_argument("--vms_api_url", "-v" help="Salient VMS API Endpoint.", required=True)
```

The `vms` variable name will have to be changed in other lines of code.